sim/constants.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Simulator constants
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * @license MIT
+ */
+
+const k = {
+	EFFECTIVENESS: {
+		INVALID: 'na',
+		NEUTRAL: 'e',
+		NO_EFFECT: 'ne',
+		NOT_VERY: 'nve',
+		SUPER: 'se',
```

Comment:
This is not what I mean by string-enum.

I mean like,

```
type MoveEffectiveness = 'invalid' | 'neutral' | 'immune' | 'not-very-effective' | 'super-effective';
```

Reply:
I think string-enum is the way to go here.